src/modify_policies.test.js

```diff
@@ -84,10 +84,20 @@ describe('Change Policies', () => {
     // waiting for the queue list to be empty and the executed tx to be on the history tab
     await assertElementPresent({ selector: transactionsTab.no_tx_in_queue, type: 'css' }, gnosisPage)
     await clickByText('button > span > p', 'History', gnosisPage)
-    await gnosisPage.waitForTimeout(2000)
+    await gnosisPage.waitForFunction(
```

Comment:
There is a shorter function to do this. We are already doing this in the rejectTx as I once found that the test was marked as completed because it happened to be another reject tx but we didn't check the nonce. You can do it like this

https://github.com/gnosis/safe-react-e2e-tests/blob/5f54cf81ef0c19513a11956fd12b3452c56c2f50/utils/actions/rejectNextTx.js#L64
```
// We wait until tx with expected nonce get into the history list
    await gnosisPage.waitForFunction((nonce) =>
      document.querySelector('div.tx-nonce').innerText.includes(nonce), {}, firstTransactionNonce
    )
```
You can still pass the selector as argument to do it more flexible like this
```
// We wait until tx with expected nonce get into the history list
    await gnosisPage.waitForFunction((selector, nonce) =>
      document.querySelector('div.tx-nonce').innerText.includes(nonce), {}, transactionsTab.tx_nonce, firsTransactionNonce
    )
```